core/src/main/java/com/datastrato/gravitino/catalog/CatalogManager.java

```diff
@@ -581,6 +593,15 @@ private String buildConfPath(String provider) {
     return String.join(File.separator, gravitinoHome, "catalogs", provider, "conf");
   }
 
+  private Optional<String> buildCustomCatalogOperationPath(Map<String, String> catalogProperties) {
+    String className = catalogProperties.get(Catalog.CATALOG_OPERATION_CLASS_NAME);
+    String path = catalogProperties.get(Catalog.CATALOG_OPERATION_CLASS_PATH);
+    if (StringUtils.isNotBlank(className) && StringUtils.isNotBlank(path)) {
```

Comment:
Could the `path` have the default value?

Reply:
the default action is not adding extra path to catalog classpaths.

Reply:
code removed